assets/css/login.less

```diff
@@ -85,21 +97,21 @@ body {
 .section2 {
     height: 250px;
     align-content: center;
-    padding-top: 50px;
-    padding-bottom: 50px;
-    padding-left: 500px;
-    padding-right: 500px;
+    padding: 50px 500px 50px 500px;    
```

Comment:
a 500px padding seems rather excessive... but nevertheless, is there any shorter way to write this property?

Reply:
Yep, there is padding: 50px 500px;